_format/1.1/index.md

```diff
@@ -1781,6 +1807,122 @@ parameter from this specification, it **MUST** return `400 Bad Request`.
+  conforms to a profile.  Thus, clients SHOULD treat profile URIs as
+  identifiers and not as links, but profiles MAY be defined in a way
+  that the URIs do identify retrievable profile description and thus
+  can be accessed by clients by dereferencing the profile URI.  For
+  profiles intended for use in environments where clients may encounter
+  unknown profile URIs, profile maintainers SHOULD consider to make the
+  profile URI dereferencable and provide useful documentation at that
+  URI.  The design and representation of such profile descriptions,
+  however, is outside the scope of this specification.
+
+A profile **MAY** assign meaning to particular elements of the document
+structure, such as resource attributes or members of meta objects, and even
+reserved query parameters, such as `filter`. The scope of a profile **MUST** be
+clearly delineated. The elements reserved by a profile, and the meaning assigned
+to those elements, **MUST NOT** change over time or else the profile **MUST** be
+considered a new profile with a new URI. However, a profile **MAY** evolve
+additively within the scope originally claimed by the profile.
+
+For example, let's say that a profile reserves a `timestamps` member in the
+`meta` object of every resource. Originally, this profile defines the value
+of `timestamps` as an object that must contain one member: `created`. The
+profile could evolve to allow an optional member, `updated`, in the `timestamps`
+object. But it could not make that member required, nor could it introduce
+a new sibling to `timestamps`.
+
+### <a href="#profile-keywords" id="profile-keywords" class="headerlink"></a> Profile Keywords
+
+A profile **SHOULD** explicitly declare "keywords" for any elements that it
+introduces to the document structure. If a profile does not explicitly declare a
+keyword for an element, then the name of the element itself can be considered a
+keyword. Keywords **MAY** be aliased in any representation through the use of
+[profile descriptors](profile-descriptors), as described below.
+
+### <a href="#profile-media-type-parameter" id="profile-media-type-parameter" class="headerlink"></a> `profile` Media Type Parameter
+
+The `profile` media type parameter is used to describe the application of
+one or more profiles to the JSON API media type. The value of the `profile`
+parameter **MUST** equal a whitespace-separated list of profile URIs.
+
+Clients and servers **SHOULD** use the `profile` media type parameter in
+conjunction with the JSON API media type in a `Content-Type` header to specify
+the application of one or more profiles to a JSON API document.
+
+A client **MAY** use the `profile` media type parameter in conjunction with the
+JSON API media type in an `Accept` header to _request_, but not _require_, that
+the server apply one or more profiles to the response. When such a request is
+received, a server **SHOULD** attempt to apply the requested profiles in
```

Comment:
How would a server signal that it did or did not succeed in this attempt?

Reply:
Profiles requested via `Accept` are considered optional. A server may apply a different set of profiles, or a limited set, to the response document. The actual applied profiles can be inspected by the client in the `Content-Type` header of the response.

If a client _requires_ that a profile be applied, then that profile should be requested via the `profile` query parameter. The server will hard error if it can't apply all the required profiles. Profiles that affect processing rules, and not just representation, should be required this way.